static/app/components/profiling/flamegraphZoomView.tsx

```diff
@@ -720,6 +722,21 @@ function FlamegraphZoomView({
     ]);
   }, [canvasPoolManager, flamegraph, scheduler]);
 
+  const handleCopyFunctionName = useCallback(() => {
+    if (!hoveredNodeOnContextMenuOpen.current) {
+      return;
+    }
+
+    navigator.clipboard
+      .writeText(hoveredNodeOnContextMenuOpen.current.frame.name)
+      .then(() => {
+        IndicatorStore.addSuccess(t('Function name copied to clipboard'));
+      })
+      .catch(() => {
+        IndicatorStore.addError(t('Failed to copy function name to clipboard'));
+      });
```

Comment:
I believe we should be using the action creators in https://github.com/getsentry/sentry/blob/master/static/app/actionCreators/indicator.tsx instead of `IndicatorStore` directly

Reply:
@Zylphrex ah, yes